table_foosball/table_foosball_01.rb

```diff
@@ -0,0 +1,141 @@
+require "securerandom"
```

Comment:
Another general comment to your style is that you should spell out names and variables. This is more standard Ruby style, and if it's too long, it's usually a smell your method or variable is trying to do too much.